lib/client/api.go

```diff
@@ -1002,8 +1006,11 @@ func (tc *TeleportClient) Logout() error {
 }
 
 // Login logs the user into a Teleport cluster by talking to a Teleport proxy.
-// If successful, saves the received session keys into the local keystore for future use.
-func (tc *TeleportClient) Login() (*CertAuthMethod, error) {
+//
+// If 'activateKey' is true, saves the received session cert into the local
+// keystore (and into the ssh-agent) for future use.
+//
+func (tc *TeleportClient) Login(activateKey bool) (*Key, error) {
```

Comment:
why would anyone would like not to add the key to the local store? I wonder if we can get rid of this flag and make this always true.